lib/cocoapods/command/spec/create.rb

```diff
@@ -60,7 +60,7 @@ def default_data_for_template(name)
           data[:author_email]  = `git config --get user.email`.strip
           data[:source_url]    = "http://EXAMPLE/#{name}.git"
           data[:ref_type]      = ':tag'
-          data[:ref]           = '#{s.version}'
+          data[:ref]           = '#{spec.version}'
```

Comment:
where does the `s` vs `spec` get denied here?

Reply:
What do you mean?

Reply:
where is the `spec` variable or method defined?

Reply:
Here: https://github.com/CocoaPods/CocoaPods/pull/7810/files#diff-a2a99835a4e1cbdefeaa4a1967e117d4R119

Reply:
oh, it's not code, its a string! I get it now